tests/languages/clojure/keyword_feature.test

```diff
@@ -0,0 +1,37 @@
+(def)
+(defmacro)
+(if)
+(do)
+(let)
+(var)
+(fn)
+
```

Comment:
Ideally, we should add checks for each and every keyword here.

Reply:
I added about 70 new keywords here - a lot of the ones that were posing issues when I was testing e.g. `<=` and `>=`.  I am happy to fill out the rest, but I have a high degree of confidence that the ones I did not include will pass ;)